sklearn/manifold/_mds.py

```diff
@@ -84,7 +83,7 @@ def _smacof_single(
     stress : float
         The final value of the stress (sum of squared distance of the
         disparities and the distances for all constrained points).
-        If `normalized_stress=True`, and `metric=False` returns Stress-1.
+        If `normalized_stress=True`, returns Stress-1.
```

Comment:
The following lines in the user guide also need to be updated accordingly:

https://github.com/scikit-learn/scikit-learn/blob/f4ba17bf88af25675c997400559f9e2e4c1cbe41/doc/modules/manifold.rst?plain=1#L479-L481

Reply:
Thanks, done.